app/models/cart.rb

```diff
@@ -13,6 +13,8 @@ def add_item(item_id)
   def minus_item(item_id)
     if contents[item_id.to_s] > 0
       contents[item_id.to_s] -= 1
+    else contents[item_id.to_s] == 0
+      contents.delete_if {|key,value| value == contents[item_id.to_s]}
```

Comment:
Unused block argument - `key`. If it's necessary, use `_` or `_key` as an argument name to indicate that it won't be used.<br>Inconsistent indentation detected.<br>Space missing after comma.<br>Space between { and | missing.<br>Space missing inside }.